time_interp/time_interp_external2.F90

```diff
@@ -126,7 +126,7 @@ module time_interp_external2_mod
     !> Holds filename and file object
     !> @ingroup time_interp_external2_mod
     type, private :: filetype
-        character(len=128) :: filename = ''
+        character(len=FMS_PATH_LEN) :: filename = ''
```

Comment:
This is a filename, shouldn't it be limited using FMS_FILE_LEN?  If so, need to change the platform module use statement as well.

Reply:
Yeah I think this one should be a file length, fixed with f9cac03